acceptance/README.md

```diff
@@ -145,6 +145,33 @@ There also may be scenarios where you want to specify the host(s) to release. E.
     HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
     HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
 
+### Running tagged tests
+
+Tests runs can be segmented by tags defined in the tests themselves. These
+tags have been standardized into the following.
+
+* risk:high
+* risk:medium
+* risk:low
```

Comment:
@glennsarti you have brought up a requirement that @MosesMendoza called to my attention as well. Your concern about new tests will be addressed by adding any tests that have changes to them dynamically to the per commit runs.

I would argue against defaulting untagged tests to the per commit runs. The purpose of the tiering workflow is to make sure the cost of execution for frequent CI runs remains as low as possible. I agree that there is risk in having untagged tests, but most new tests should not jump into the per commit runs.

Automatically running changed (which would include new) tests in the per commit run should mitigate this risk. I am also interested in your thoughts about adding a commit hook to ensure that committed tests contain a tag.